tomviz/python/Auto_Correlation.py

```diff
@@ -0,0 +1,18 @@
+def transform_scalars(dataset):
+    """Apply auto correlation on a dataset."""
+    
+    from tomviz import utils
+    from scipy.fftpack import fftn, ifftn, fftshift
+    import numpy as np
+    
+    array = utils.get_array(dataset)
+    dim = []
```

Comment:
This is not necessary as ```dim``` is assigned at line 10.

Reply:
The command in the line after, array.shape, gives me a tuple when I want create a numpy array.

Reply:
Correct, so you can delete line 9.